frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -36,7 +36,7 @@ import { EmbedModalContentStatusBar } from "./EmbedModalContentStatusBar";
 import { OverviewSettings } from "./OverviewSettings";
 import { ParametersSettings } from "./ParametersSettings";
 import { PreviewModeSelector } from "./PreviewModeSelector";
-import { PreviewPane } from "./PreviewPane";
+import { PreviewPane } from "./PreviewPane/PreviewPane";
```

Comment:
We need to add `PreviewPane/index.ts` 

```suggestion
import { PreviewPane } from "./PreviewPane";
```